src/components/Header.js

```diff
@@ -120,10 +123,10 @@ function NavBar(){
     )
 }
 
-export default function Header(){
+export default function Header({change}){
     return(
         <MainHeader>
-            <NavBar></NavBar>
+            <NavBar action={change}></NavBar>
```

Comment:
You could put a flag here to refactor when you learn how to use useContext to avoid prop drilling :)
When using props, is a good practice to ensure all the components that received that props really need them, otherwise composition or hooks can help us, but at this point of the Bootcamp its okay